src/nv_ingest/extraction_workflows/pdf/pdfium_helper.py

```diff
@@ -389,16 +412,19 @@ def pdfium_extractor(
                 extracted_data.extend(image_data)
 
             # If we want tables or charts, rasterize the page and store it
-            if extract_tables or extract_charts:
-                image, _ = pdfium_pages_to_numpy(
-                    [page], scale_tuple=(YOLOX_MAX_WIDTH, YOLOX_MAX_HEIGHT), trace_info=trace_info
+            if extract_tables or extract_charts or extract_infographics:
+                image, padding_offsets = pdfium_pages_to_numpy(
+                    [page],
+                    scale_tuple=(YOLOX_PAGE_IMAGE_PREPROC_WIDTH, YOLOX_PAGE_IMAGE_PREPROC_HEIGHT),
+                    padding_tuple=(YOLOX_PAGE_IMAGE_PREPROC_WIDTH, YOLOX_PAGE_IMAGE_PREPROC_HEIGHT),
```

Comment:
Padding is necessary here to align with the above comment regarding the bounding boxes in metadata: https://github.com/NVIDIA/nv-ingest/pull/483/files#r1970386230